rbac/admin.py

```
@@ -0,0 +1,603 @@
+    ]
+    
+    readonly_fields = [
+        'id', 'access_token', 'refresh_token',
+        'created_at', 'last_activity', 'is_expired'
+    ]
+    
+    fieldsets = (
+        ('کاربر', {
+            'fields': ('user',)
+        }),
+        ('توکن‌ها', {
```

Ruff (0.12.2)

String contains ambiguous `ه` (ARABIC LETTER HEH). Did you mean `o` (LATIN SMALL LETTER O)?

(RUF001)